Sources/GRButton/GRButtonViewModel.swift

```diff
@@ -0,0 +1,437 @@
+//
+//  sourcery_iOS
+//
+//  Created by Andrej Jasso on 28/10/2022.
+//
+
+import SwiftUI
+
+public struct GRButtonViewModel {
+    
+    public let appearance: AppearanceModel
+    
+    public enum StyleType {
+        
+        case base
+        case sampleBase
+        case sampleText
+        case destructive
+        case text
+        case textDestructive
+        case sourcery
+        
+        case circularPrimary
+        case circularSecondary
+        
+        case social
+        case socialSquare
+        
+        case custom(AppearanceModel)
```

Comment:
Tie nazvy stylov mi pridu celkom nepopisne. Aky je rozdiel mediz base a sample base. Text a sampleText? Chcelo by to premenovat.

Reply:
Ako pozeram figmu, vacsinu z nich ani netreba, vymazavam